src/test/java/de/tum/in/www1/artemis/ExampleSubmissionIntegrationTest.java

```diff
@@ -243,4 +250,73 @@ public void createExampleTextAssessment_wrongExerciseId() throws Exception {
     }
 
+    private ExampleSubmission importExampleSubmission(HttpStatus expectedStatus, Long submissionId, Long exerciseId) throws Exception {
+        return request.postWithResponseBody("/api/exercises/" + exerciseId + "/example-submissions/import", submissionId, ExampleSubmission.class, expectedStatus);
+    }
+
+    @Test
+    @WithMockUser(value = "instructor1", roles = "INSTRUCTOR")
+    public void importExampleSubmissionWithTextSubmission() throws Exception {
+        Submission submission = ModelFactory.generateTextSubmission("submissionText", Language.ENGLISH, true);
+        submission = database.saveTextSubmission(textExercise, (TextSubmission) submission, "student1");
+
+        TextBlock textBlock = new TextBlock();
+        textBlock.setCluster(null);
+        textBlock.setAddedDistance(0);
+        textBlock.setStartIndex(0);
+        textBlock.setEndIndex(14);
+        database.addAndSaveTextBlocksToTextSubmission(Set.of(textBlock), (TextSubmission) submission);
+
+        database.addResultToSubmission(submission, AssessmentType.MANUAL);
+
+        // add one feedback for the created text block
+        List<TextBlock> textBlocks = new ArrayList<>(((TextSubmission) submission).getBlocks());
+        Feedback feedback = new Feedback();
+        if (!textBlocks.isEmpty()) {
+            feedback.setCredits(1.0);
+            feedback.setReference(textBlocks.get(0).getId());
+        }
+
+        database.addFeedbackToResult(feedback, submission.getLatestResult());
+
+        ExampleSubmission exampleSubmission = importExampleSubmission(HttpStatus.OK, submission.getId(), textExercise.getId());
+        List<TextBlock> copiedTextBlocks = new ArrayList<>(((TextSubmission) exampleSubmission.getSubmission()).getBlocks());
+        assertThat(exampleSubmission.getId()).isNotNull();
+        assertThat(((TextSubmission) exampleSubmission.getSubmission()).getText()).isEqualTo(((TextSubmission) submission).getText());
+        assertThat(exampleSubmission.getSubmission().getLatestResult().getFeedbacks().size()).isEqualTo(1);
+        assertThat(copiedTextBlocks.size()).isEqualTo(1);
+        assertThat(exampleSubmission.getSubmission().getLatestResult().getFeedbacks().get(0).getReference()).isEqualTo(copiedTextBlocks.get(0).getId());
+    }
+
+    @Test
+    @WithMockUser(value = "instructor1", roles = "INSTRUCTOR")
+    public void importExampleSubmissionWithModelingSubmission() throws Exception {
+        Submission submission = ModelFactory.generateModelingSubmission(validModel, true);
+        submission = database.addModelingSubmission(modelingExercise, (ModelingSubmission) submission, "student1");
+        database.addResultToSubmission(submission, AssessmentType.MANUAL);
+
+        ExampleSubmission exampleSubmission = importExampleSubmission(HttpStatus.OK, submission.getId(), modelingExercise.getId());
+        assertThat(exampleSubmission.getId()).isNotNull();
```

Comment:
Please add a check to assert equality between the modelling strings as well (as with the text in the text exercise)